app/services/trainees/create_from_apply.rb

```diff
@@ -41,12 +41,8 @@ def mapped_attributes
         ethnic_background: raw_trainee["ethnic_background"],
         diversity_disclosure: diversity_disclosure,
         email: raw_contact_details["email"],
-        training_route: course&.route,
```

Comment:
I decided to only copy the course code and route because `Trainees::ApplyRegistrations::ConfirmCoursesController` and `ApplyRegistrations::ConfirmCourseForm` already copy all the fields when the user confirms. By the leaving the other fields blank it makes it easier for `Trainees::ApplyRegistrations::CourseDetailsController` to check whether the can go straight to the confirmation page or not.